### PR TITLE
[travis] Cache what's possible to speed up the build

### PR DESCRIPTION
**Pull request by rdzhabarov**
*Description*:
Mostly time is saved on utils/install_protobuf.sh run (~10%).
Going to merge this and then CircleCI might give us some wins later on.

*Testing*:
CI

*Documentation*:
n/a

### CODE REVIEW

File: lib/Importer/ONNXIFIModelLoader.cpp [hunk 1]
```diff
@@ -137,7 +137,8 @@ std::unique_ptr<ONNXIFIModelLoader> ONNXIFIModelLoader::parse(
 }
 
 std::vector<std::pair<Kinded::Kind, ElemKind>>
-ONNXIFIModelLoader::parseOperator(const void *onnxModel, size_t onnxModelSize) {
+ONNXIFIModelLoader::parseOperators(const void *onnxModel,
```
Comment (rdzhabarov) [inline lib/Importer/ONNXIFIModelLoader.cpp]: some unrelated changes. will revert.

**Comment by bertmaher** [comment]
shipitwithfire
^^^ see, this is why I miss Phabricator macros!
